bitstream.h

```diff
@@ -34,21 +34,25 @@ class cBitstream
   int      m_offset;
   int      m_len;
   bool     m_error;
+  bool     m_doEP3;
```

Comment:
why did you not override cBitstream, i.e.
class cBitstreamEP3 : public cBitstream